packages/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -122,6 +131,11 @@ export default defineComponent({
     DatePicker: () => import('v-calendar/lib/components/date-picker.umd')
   },
   props: {
+    calendarIcon: {
```

Comment:
Can we just name this icon since it’s the only one? 

```suggestion
    icon: {
```

Reply:
yes, we can :)